docs/effects/lighting-effect.md

```diff
@@ -0,0 +1,114 @@
+# LightingEffect
+
+The Lighting Effect applies ambient, point and directional lighting to layers which support material property.
+
+```js
+import DeckGL, {GeoJsonLayer, LightingEffect} from '@deck.gl/core';
+import {AmbientLight, PointLight, DirectionalLight, PhongMaterial} from '@luma.gl/core';
+
+// create ambient light source
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 1.0
+});
+// create point light source
+const pointLight = new PointLight({
+  color: [255, 255, 255],
+  intensity: 2.0,
+  // use coordinate system as the same as view state
+  position: [-125, 50.5, 5000]
+});
+// create directional light source
+const directionalLight = new DirectionalLight({
+  color: [255, 255, 255],
+  intensity: 1.0,
+  direction: [-3, -9, -1]
+});
+
+const lightingEffect = new LightingEffect({ambientLight, pointLight, directionalLight});
+
+const material =  new PhongMaterial({
+  ambient: 0.2,
+  diffuse: 0.5,
+  shininess: 32,
+  specularColor: [255, 255, 255]
+});
+
+const INITIAL_VIEW_STATE = {
```

Comment:
No need to emphasize this. Since you are omitting geojson data in this code sample, you can skip irrelevant deck props too.

Reply:
The reason I keep this view state here is that, I want to show users the coordinate system in point lights is the same as view state

Reply:
Add a comment then?

Reply:
comment is added in point light position line, don't want to duplicate here.